test/support/utils_sycl.h

```diff
@@ -137,7 +137,7 @@ sycl::queue get_test_queue()
     return my_queue;
 }
 
-template <sycl::usm::alloc alloc_type, typename TestValueType, typename TestName>
+template <sycl::usm::alloc alloc_type, typename TestValueType, typename TestName, size_t kStartIndex = 1>
```

Comment:
Why do wee need such default value?
In my understanding each test should run algo on a range [0, max_n].

Reply:
I am tried to do this variant. But many changes required for this.
So I think that we can test from 0 only these tests, which are really required.